hpobench/container/recipes/mo/Singularity.LanguageModelBenchmark

```diff
@@ -0,0 +1,30 @@
+Bootstrap: docker
+From: python:3.7-slim
+
+%labels
+MAINTAINER sharmaa@informatik.uni-freiburg.de
+VERSION v0.0.1
+
+%post
+    apt update -y
+    apt install build-essential git wget -y
+
+    cd /home \
+    && mkdir data && cd data \
+    && wget https://raw.githubusercontent.com/pytorch/examples/master/word_language_model/data/wikitext-2/train.txt \
```

Comment:
no need to download in container. The dataloader should download the data automatically to the users host system (defined data dir in the hpobenchrc file)